src/fanControl.ino

```diff
@@ -0,0 +1,279 @@
+/*************************************************** 
+  This is an example for our Adafruit 16-channel PWM & Servo driver
+  PWM test - this will drive 16 PWMs in a 'wave'
+  Pick one up today in the adafruit shop!
+  ------> http://www.adafruit.com/products/815
+  These drivers use I2C to communicate, 2 pins are required to  
+  interface.
+  Adafruit invests time and resources providing this open source code, 
+  please support Adafruit and open-source hardware by purchasing 
+  products from Adafruit!
+  Written by Limor Fried/Ladyada for Adafruit Industries.  
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <Wire.h>
+#include <Adafruit_PWMServoDriver.h>
+
+// called this way, it uses the default address 0x40
+Adafruit_PWMServoDriver pwm = Adafruit_PWMServoDriver();
+// you can also call it with a different address you want
+//Adafruit_PWMServoDriver pwm = Adafruit_PWMServoDriver(0x41);
+// you can also call it with a different address and I2C interface
+//Adafruit_PWMServoDriver pwm = Adafruit_PWMServoDriver(0x40, Wire);
+
+/*
+Command structure:
+Byte 3:  Command ID
+Byte 2:  parameter 1
+Byte 1:  Parameter 2
+Byte 0:  Response, command response status
+
+The Arduino responds to every command with 4 bytes, Command ID, parameter 1 & 2 and CMD_OK/COMMAND_ERROR
+*/
+
+#define FAN_ID 5

```

Comment:
Is `FAN_ID` some static ID for the arduino? it's used every time the command is `CMD_FAN_ID`. Is this a way to ensure that the serial port we opened is to the arduino we want, and not some other/additional arduino connected to the same PC? If so, maybe something other than 5 is better, something a bit more "unique", like 197 or something...

Reply:
Legacy. That's your old command from when you needed to find out which fan array you were talking to.